lib/paperclip/processors/gzip.rb

```diff
@@ -15,9 +15,11 @@ def make
         dst = create_tempfile
         begin
           gz = Zlib::GzipWriter.new(dst, level, strategy)
-          gz.write(@file.read)
-        ensure
+          gz.write @file.read
           gz.close
+        rescue ::Exception => e
```

Comment:
Avoid rescuing the `Exception` class. Perhaps you meant to rescue `StandardError`?